package.json

```diff
@@ -28,6 +28,7 @@
   "standard": {
     "parser": "babel-eslint",
     "ignore": [
+      "packages/next-server/server/next-server.ts",
```

Comment:
The standardjs/eslint parser for typescript couldn't parse the file. So it's time to implement `tslint`. I'll create a good first issue for it after this PR is merged

Reply:
You can create a separate override section with typescript-eslint-parser for .ts, .tsx files

Reply:
It's already using the ts parser, it's just not parsing public/private correctly